nimbus-features/fakespotFeature.yaml

```diff
@@ -27,9 +33,11 @@ features:
       - channel: beta
         value:
           status: true
+          product-recommendations: true
```

Comment:
This should be turned off for now. Only when all the views and API calls are in place we should turn this on.